src/main/kotlin/com/bridgecrew/ui/buttons/CheckovLinkButton.kt

```diff
@@ -0,0 +1,19 @@
+package com.bridgecrew.ui.buttons
+
+import java.awt.Cursor
+import javax.swing.JButton
+
+open class CheckovLinkButton(private var text: String): JButton() {
```

Comment:
I think that the Link name causes to think it's leading to a url or something like that, maybe we can use a different one like "CheckovResultActionButton"? Or something better 😄

Reply:
The idea was to create a UI component that looks like a link and behaves like a button.
In the documentation case - it's indeed a link (URL)